### PR TITLE
Update get-started.md

### PR DESCRIPTION
**Pull request by kwannoel**
## Info

- **Description**

  - [ What's changed? Which parts of the docs are affected? ]

- **Notes**

  - [ Include any supplementary context or references here. ]

- **Related code PR**

  - [ Provide a link to the relevant code PR here, if applicable. ]

- **Related doc issue**
  
  Resolves [ Provide a link to the relevant doc issue here, if applicable. ]

## For reviewers

- **Preview**

  - [ Paste the preview link to the updated page(s) here. Edit this item after the preview site is ready. To find the updated pages, scroll down to locate and open the Amplify preview link and select the **dev** version of the documentation. ]

- **Key points**

  - [ Parts that may need revision or extra consideration. ]

## Before merging

- [ ] I have checked the doc site preview, and the updated parts look good.

- [ ] I have acquired the approval from the owner (and optionally the reviewers) of the code PR and at least one tech writer (`CharlieSYH`, `emile-00`, & `hengm3467`).

### CODE REVIEW

File: versioned_docs/version-1.7/get-started.md [hunk 1]
```diff
@@ -21,6 +21,8 @@ For extensive testing or single-machine deployment, consider [starting RisingWav
 
 :::caution
 There is a known limitation when using the standalone mode on macOS. Connectors for `jdbc`, `postgresql-cdc`, `mysql-cdc`, `elastic-search`, and `cassandra` may not function as expected. We anticipate that this limitation will be addressed in version 1.8.
+
+You will also need a machine with `AVX2` instruction set support to run RisingWave.
```
Comment (hengm3467) [inline versioned_docs/version-1.7/get-started.md:25]: @kwannoel I feel this note is a bit too technical. I googled this instruction set and wasn't sure if my laptop is included. This note may confuse people more than it clarified the matter.

Comment (kwannoel) [reply versioned_docs/version-1.7/get-started.md:25]: I wanted to add it inside troubleshooting instead. But seems it is for kubernetes

Screenshot 2024-03-11 at 1 19 49 PM

Comment (hengm3467) [reply versioned_docs/version-1.7/get-started.md:25]: @kwannoel Thanks for bringing this up! 
@ShanlanLi Please rework the troubleshooting landing page to reflect that this is for not K8s only. You can also evaluate if we need to restructure the content in this topic (like creating separate topics). Thanks.

Comment (ShanlanLi) [reply versioned_docs/version-1.7/get-started.md:25]: OK,will do it today

**Comment by ShanlanLi** [comment]
This has been resoved by https://github.com/risingwavelabs/risingwave-docs/pull/1987#event-12383138256

**Pull request closed without merging**